### DISCUSSION TITLE
Text isn't visible when I upload a svg image in WordPress site!

### DISCUSSION THREAD

**Discussion by rajatshubhra** [jgraph/drawio#4238] [General]
I used to draw diagrams using draw.io for my wordpress site. But recently in exported image text isn't visible when I upload that svg image in WordPress site. Embed fonts option was checked before exporting! Any solution?

**Comment by davidjgraph** [comment]
Does it work if you open the SVG directly in a browser?

**Comment by slawr** [comment]
If it helps anyone, try unchecking embed fonts and rechecking before export. See https://github.com/jgraph/drawio/issues/4758
